### ISSUE TITLE
Add MIDI module to 0.6.0 build

### ISSUE THREAD

**Issue by GregBro** [gratrix/vcv-gratrix#8]
Bernd Freundorfer asked me to open an issue as a reminder to add the 6 channel MIDI module to the 0.6.0 build.

**Comment by gratrix** [comment]
Yep will be the first thing I do when I get time to work on my modules again.

**Comment by michaeljblum** [comment]
Hi Gratrix, just checking in about this matter. It seems that the MIDI modules are still missing from 0.6.0, even though their SVG files seem to be present in the directory. Would be greatly appreciated if you could add them in completely. Thanks.

**Comment by markHadman** [comment]
Do we know what the underlying issue is here? 
Just a flag in the code that we could fork and fix?

**Comment by dcbb** [comment]
Would love that 6 channel MIDI module, too!

**Comment by gratrix** [comment]
Just seen polyphonic cables are to be in v1.0 which I presume makes most of my modules redundant! I'm cool with this as it's clearly a better approach :) I'm glad to have been able to make polyphonic modular available in the meantime.
